src/KBprojectorSparse.h

```diff
@@ -30,7 +30,7 @@
 class KBprojectorSparse : public KBprojector
 {
     // work arrays (1 for each thread)
-    static std::vector<std::vector<ORBDTYPE>> work_nlindex_;
+    static std::vector<std::vector<double>> work_nlindex_;
```

Comment:
Could we use KBPROJDTYPE instead of fixing it to double? Could there be a chance we would want to use low-precision there? Or perhaps high precision arithmetic on low precision data, since incoming orbitals are probably already in low precision so we can avoid casting the data?

Reply:
fixed